iopipe/constants.py

```diff
@@ -4,6 +4,6 @@
 
 COLDSTART = True
 MODULE_LOAD_TIME = time.time() * 1000
-PROCESS_ID = os.getpid()
+PROCESS_ID = str(os.getpid())
```

Comment:
PROCESS_ID is used by our own tracing to uniquely id processes, and should be a UUID: https://github.com/iopipe/iopipe/blob/master/src/globals.js#L9, not from the OS

Reply:
Ok, fixed.